applications/cm-service/values-usdf-cm.yaml

```diff
@@ -0,0 +1,31 @@
+config:
+  logLevel: "INFO"
+  logProfile: "development"
+  databaseEcho: true
+  outputVolume:
+    storageClassName: "sdf-data-rubin"
+    subPath: "shared/campaigns/users/usdf-cm-prod"
+worker:
+  htcondor:
+    config:
+      mountPath: "/opt/lsst/software/stack/conda/envs/lsst-scipipe-9.0.0/etc/condor/config.d"
```

Comment:
Suspicious looking path; is rubin-env actually mounted there?

Reply:
Yup. This nests our condor config within the config defaults that come along with the stack.  Otherwise, setting up the stack in the container wipes out our config when it repoints `CONDOR_CONFIG`.